app/src/components/fineTunes/FineTuneContentTabs/General/InferenceCodeTabs/InferenceCodeTabs.tsx

```diff
@@ -0,0 +1,131 @@
+import { useMemo } from "react";
+
+import { type RouterOutputs } from "~/utils/api";
+import { useFineTune, useSelectedProject, useTrainingEntries } from "~/utils/hooks";
+import CopiableCodeTabs, { type CodeTab } from "./CopiableCodeTabs";
+
+const baseTabs: CodeTab[] = [
+  {
+    title: "cURL",
+    language: "bash",
+    code: `curl --request POST \\
+--url https://app.openpipe.ai/api/v1/chat/completions \\
+--header 'Authorization: Bearer {{TEMPLATED_OPENPIPE_API_KEY}}' \\
+--header 'Content-Type: application/json' \\
+--data '{
+"model": "openpipe:{{FINE_TUNE_MODEL_SLUG}}",
+"messages": {{TEMPLATED_MESSAGES}},
+"tool_choice": {{TEMPLATED_TOOL_CHOICE}},
+"tools": {{TEMPLATED_TOOLS}}
+}'`,
+  },
+  {
+    title: "Python",
+    language: "python",
+    code: `from openpipe import OpenAI
```

Comment:
Let's add a comment to the top for both this and the JS one where we explain how to install the library. Eg. `# pip install openpipe`